www/apps/resources/app/commerce-modules/auth/auth-flows/page.mdx

```diff
@@ -149,3 +149,32 @@ if (success) {
 If the returned `success` property is `true`, the authentication with the third-party provider was successful.
 
 ![Diagram showcasing the second part of the third-party authentication flow](https://res.cloudinary.com/dza7lstvk/image/upload/v1711375123/Medusa%20Resources/third-party-auth-2_kmjxju.jpg)
+
+---
+
+## Reset Password
```

Comment:
nit: It is maybe worth highlighting that this should be hidden behind an authenticated flow of events and then link to how it is implemented at the HTTP layer in our core